src/community/pom.xml

```diff
@@ -462,5 +464,17 @@
         <module>geogig</module>
       </modules>
     </profile>
+    <profile>
+      <id>oauth2</id>
```

Comment:
Why having a second profile? web/app/pom.xml only has oauth2-google in any case.